tests/integration/components/ember-power-select-test.js

```diff
@@ -1821,4 +1835,30 @@ test('the list of options can be customized using optionsComponent', function(as
   let text = $('.ember-power-select-options').text().trim();
   assert.ok(/Countries:/.test(text), 'The given component is rendered');
   assert.ok(/3\. Russia/.test(text), 'The component has access to the options');
+});
+
+
+/**
+11 - Development assertions
+  a) the `onchange` function is mandatory.
+*/
+
+moduleForComponent('ember-power-select', 'Integration | Component | Ember Power Select (Assertions)', {
+  integration: true
+});
+
+test('the `onchange` function is mandatory', function(assert) {
+  assert.expect(1);
+
+  this.numbers = numbers;
+
+  try {
+    this.render(hbs`
+      {{#ember-power-select options=countries selected=selected as |opt|}}
+        {{opt}}
+      {{/ember-power-select}}
+    `);
+  } catch(e) {
+    assert.ok(/requires an `onchange` function/.test(e.message));
```

Comment:
Qunit's `throws` function could be used in this test.
https://api.qunitjs.com/throws/

Not crucial.